packages/f-alert/src/demo/Index.vue

```diff
@@ -6,11 +6,60 @@
         <meta
             name="viewport"
             content="width=device-width, initial-scale=1">
+        <div data-test-id="component-controls">
```

Comment:
This is super cool – the tidy freak in me says that it would be good to have this as a separate `DemoControls.vue` component that gets pulled into the `index.vue` potentially?

Reply:
I was wondering about whether to reuse f-form-field and have it pulled in as a devDependency 😄

Reply:
hahaha – could do 😉